index.bs

```diff
@@ -2649,6 +2695,21 @@ Given an [=attribution trigger=] |trigger|, an [=attribution source=]
 
 <h3 algorithm id="creating-aggregatable-contributions">Creating aggregatable contributions</h3>
 
+To <dfn>match [=aggregatable contributions=] to [=attribution source/aggregation keys=]</dfn> given
```

Comment:
Wdyt about  renaming this algorithm to `create aggregatable contributions from aggregation keys and aggregatable values`?

Reply:
Sounds good to me.